spec/requests/api/bookings_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe "Api::Bookings", type: :request do
+  
+  describe 'Create Booking' do
+    request_params_success_case = {
```

Comment:
Use `let` instead of local variable https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let